exercicioLogica19/src/main/java/org/example/Agenda.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+
+import java.util.ArrayList;
+
+public class Agenda {
+
+    private Contato contato;
```

Comment:
essas 4 propriedades poderiam ser substituídas por uma única só, a propriedade que elas realmente representam, uma lista de contatos .

ex: `private List<Contato> contatos;`

Também é recomendado fazer a o "new" de uma propriedade somente quando for utiliza-la, nesse caso podendo ser feito quando fosse utilizado o método construtor da classe já que sera o momento que é criada uma nova instancia da classe Agenda.
```
public Agenda() {
        this.contatos = new ArrayList<>();
}
```